specification/synapse/data-plane/readme.typescript.md

```diff
@@ -20,4 +22,12 @@ batch:
       package-name: "@azure/synapse-spark"
       output-folder: $(typescript-sdks-folder)/sdk/synapse/synapse-spark
       clear-output-folder: true
+    - package-vnet: true
+      package-name: "@azure/synapse-managed-private-endpoints"
+      output-folder: $(typescript-sdks-folder)/sdk/synapse/synapse-managed-private-endpoints
+      clear-output-folder: true
+    - package-monitoring: true
+      package-name: "@azure/synapse-monitoring"
+      output-folder: $(typescript-sdks-folder)/sdk/synapse/synapse-monitoring
+      clear-output-folder: true
```

Comment:
Do we need to add 
```
      tracing-info.namespace: "Microsoft.Synapse" 
      tracing-info.packagePrefix: "Azure.Synapse"
```

That is a new feature from autorest.typescript  above 6.0.0-dev.20201113.1.  You can find it from our email thread with Jose.

Reply:
Please using above 6.0.0-dev.20201120.2 version to generate SDK.  Because we found serials of issues using previous version and Jose had fixed them after feedback, using latest version may avoid these known issues. Thanks a lot~